### DISCUSSION TITLE
Can't execute locally with `./webots`. No such file or directory

### DISCUSSION THREAD

**Discussion by KristofferNyvoll** [cyberbotics/webots#5427] [Q&A]
Following the Linux [Installation guide](https://github.com/cyberbotics/webots/wiki/Linux-installation/#build-webots), but I can't complete the very last step. 

After running `make` in the `dependencies` folder, I get the following output: 
```
kristoffer@kristoffer-ubuntu ~/webots/dependencies
 % make
# copying clang-format configuration file
# downloading webots-qt-6.2.3-linux64-release.tar.bz2
# uncompressing webots-qt-6.2.3-linux64-release.tar.bz2
# downloading openal-linux64-1.16.0.tar.bz2
# uncompressing openal-linux64-1.16.0.tar.bz2
# downloading libOIS.1.4.tar.bz2
# uncompressing libOIS.1.4.tar.bz2
# downloading libpico.tar.bz2
# uncompressing libpico.tar.bz2
# downloading lua-5.2.3.tar.gz
# uncompressing lua-5.2.3.tar.gz
# compiling lua
# downloading libassimp-5.2.3.tar.bz2
# uncompressing libassimp-5.2.3.tar.bz2
# downloading libssl_1.1.tar.xz
# uncompressing libssl_1.1.tar.xz
```
So that should be okay, right?

The installation guide then says that I should run `./webots` - but that exits with error: `zsh: no such file or directory: ./webots`
Tried running it in both `dependencies`and the root directory, but to no avail. I also can't see a file named "webots"

**Comment by stefaniapedrazzi** [comment]
You have to run `make` from the the root directory to build Webots.
If you ran it in the `dependencies` directory you didn't compile Webots but just installed some of the dependencies.
During the build the Webots launcher will be copied in the root directory and you will then be able to run `./webots` from the root directory.

**Comment by stefaniapedrazzi** [comment]
I just improved the instructions so that it is now clearer in which folder you have to run the `make` instruction:
https://github.com/cyberbotics/webots/wiki/Linux-installation
